### ISSUE TITLE
Question about Fig. 4 in the paper

### ISSUE THREAD

**Issue by yutaro-s** [facebookresearch/barlowtwins#48]
Hi. Thanks for the great work!
I'm trying to reproduce your paper's results in Fig. 4 (effect of the dimensionality of the last layer of the projector network on performance).
I have two questions about this:
- Could you teach me what hyperparameters you used in the experiments? 
- Did you run main.py with `--projector 8192-8192-16384`? not `--projector 16384-16384-16384`, right?

**Comment by jingli9111** [comment]
Hi @yutaro-s 

Figure 4 uses the same hyperparameters as the baseline setting. If you are tuning it on a different setting, you should just tune the learning rate and lambda.
It's 16384-16384-16384.

**Comment by yutaro-s** [comment]
Thanks for your quick reply.
Every model was trained for 300 epochs, right?

**Comment by jingli9111** [comment]
Yes

**Comment by yutaro-s** [comment]
Thank you! I'll try it.
